package.json

```diff
@@ -15,9 +15,9 @@
   "scripts": {
     "test": "mocha --inline-diffs --check-leaks --ui tdd --reporter dot --timeout 5000 --recursive test",
     "build": "babel --auxiliary-comment-before 'istanbul ignore next' --modules commonStrict --source-maps-inline --out-dir dist src",
-    "generate-coverage": "istanbul cover _mocha -- --inline-diffs --check-leaks --ui tdd --reporter dot --timeout 5000 --recursive test",
+    "generate-coverage": "istanbul cover -x '**/dist/index.js' _mocha -- --inline-diffs --check-leaks --ui tdd --reporter dot --timeout 5000 --recursive test",
```

Comment:
The file Babel generates is only covered if every exported name `require`d somewhere. Some names are made available so new generators can be written, but not imported by the test suite. See src/index.is.